client/src/components/TextArea/Text.css

```diff
@@ -1,13 +0,0 @@
-
-
-.text-area {
-    width: 400px;
-    height: 150px;
-    background-color: rgb(227,224,222);
-    color: rgb(94,91,90);
-    font-size: 25px;
-}
-
-.text-area-title {
-    font-size: 30px;
-}
```

Comment:
Did you mean to delete this file? If you merge this branch into develop and have deleted this file it will remove it from the delete branch.

Reply:
This was Dilmourads.